### ISSUE TITLE
Forum reference in Lemmy posts

### ISSUE THREAD

**Issue by Liwott** [friendica/friendica#11271]
- [X] I have searched open and closed issues for duplicates

### Bug Description

I am following several [Lemmy](https://join-lemmy.org/) communities (ie forums) from a friendica account and their posts correctly appear in my network feed. A lot of them appear without any reference to the community.

### Steps to Reproduce

1.  Paste the URL of a Lemmy post (e.g. https://feddit.de/post/92593) into Friendica's search bar
2. Click the post

Actual Result:

In all cases, the post appears as posted by their authors. In some cases, it appears as reshared by the community.
When it does not, it says "fetched because of [author or some commenter or some liker]" when hovering the logo next to the date.

Expected Result:

All posts should refer to the community.

### Screenshots

2 example by the same author on the same community. In both cases, I was already following the community before fetching the post, and I don't follow the author.

<img width="690" alt="Screenshot 2022-02-21 at 16 50 15" src="https://user-images.githubusercontent.com/66678546/154988354-622ac17e-ff6f-45ed-b5a8-86f299f5c1f7.png">
<img width="688" alt="Screenshot 2022-02-21 at 16 50 30" src="https://user-images.githubusercontent.com/66678546/154988359-91696dd5-1531-4f44-98f5-e56c86a28833.png">

### Platform Info

This is Friendica, version 2022.02 that is running at the web location https://nerdica.net. The database version is 1449/1449, the post update version is 1427/1427.

**Comment by annando** [comment]
Yeah, I experienced the same. it seems as if Lemmy is not always sharing the starting post. Then we have to refetch it. This leads to this situation. I have the feeling as if this is caused by some Lemmy behaviour. But before opening an issue at their repository, I want to perform some more logging.

**Comment by informapirata** [comment]
> before opening an issue at their repository, I want to perform some more logging.

If I've understood correctly, the problem is that Friendica is a social network, while Lemmy isn't (yet).
Friendica, like all ActivityPub-based social networks, allows you to view who has reacted to a post only if that user is your connection. You can see that your post has been re-shared by the !green@lemmy.ml community only if you already follow that community (by the way, in the post you indicated there is already an anomaly from the standard: in fact the user is a feddit user .it, but the community on which he writes is lemmy.ml!).

Unlike Friendica, Lemmy is not a social network, but it is an aggregator (with social functions) that staticizes posts addressed to a community. Lemmy is designed to highlight (obviously) which community a post has been directed to and even which other communities have shared the same link.

Therefore it is normal that Lemmy gives you visibility on this aspect.

I ask @annando  if my interpretation is correct
